### PR TITLE
Added tlg5023.tlg011

### PR DESCRIPTION
**Pull request by planatheisa**
Checking hook.

### CODE REVIEW

**Comment by planatheisa** [comment]
@sonofmun 

I can't figure out the problem here.

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `coverage`         |    -0.03 |
| `metadata_passing` |     4    |
| `metadata_total`   |     4    |
| `nodes_count`      |   101    |
| `texts_passing`    |     1    |
| `texts_total`      |     2    |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +3467 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg2136/__cts__.xml`                          | New      |
| `./data/tlg2136/tlg001/__cts__.xml`                   | New      |
| `./data/tlg2136/tlg001/tlg2136.tlg001.opp-grc1.xml`   | New      |
| `./data/tlg5023/__cts__.xml`                          | New      |
| `./data/tlg5023/tlg011/__cts__.xml`                   | New      |
| `./data/tlg5023/tlg011/tlg5023.tlg011.1st1K-grc1.xml` | New      |
| `./data/tlg2136/__cts__.xml`                          | Passing  |
| `./data/tlg2136/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg2136/tlg001/tlg2136.tlg001.opp-grc1.xml`   | Passing  |
| `./data/tlg5023/__cts__.xml`                          | Passing  |
| `./data/tlg5023/tlg011/__cts__.xml`                   | Passing  |
| `./data/tlg5023/tlg011/tlg5023.tlg011.1st1K-grc1.xml` | Failing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3378)*

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +4 |
| `metadata_total`   |       +4 |
| `nodes_count`      |     +203 |
| `texts_passing`    |       +2 |
| `texts_total`      |       +2 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +7396 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg2136/__cts__.xml`                          | New      |
| `./data/tlg2136/tlg001/__cts__.xml`                   | New      |
| `./data/tlg2136/tlg001/tlg2136.tlg001.opp-grc1.xml`   | New      |
| `./data/tlg5023/__cts__.xml`                          | New      |
| `./data/tlg5023/tlg011/__cts__.xml`                   | New      |
| `./data/tlg5023/tlg011/tlg5023.tlg011.1st1K-grc1.xml` | New      |
| `./data/tlg2136/__cts__.xml`                          | Passing  |
| `./data/tlg2136/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg2136/tlg001/tlg2136.tlg001.opp-grc1.xml`   | Passing  |
| `./data/tlg5023/__cts__.xml`                          | Passing  |
| `./data/tlg5023/tlg011/__cts__.xml`                   | Passing  |
| `./data/tlg5023/tlg011/tlg5023.tlg011.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3381)*

**Comment by planatheisa** [comment]
Passing now.

File: data/tlg5023/__cts__.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<ti:textgroup xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:greekLit:tlg5023">
+  <ti:groupname xml:lang="lat">Scholia Graeca</ti:groupname>
```
Comment (sonofmun) [inline data/tlg5023/__cts__.xml]: Change author to "Scholia in Euripidem"

File: data/tlg5023/tlg011/__cts__.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg5023" xml:lang="grc" urn="urn:cts:greekLit:tlg5023.tlg011">
+    <ti:title xml:lang="lat">Scholia in Euripidis Rhesum</ti:title>
+  <ti:edition urn="urn:cts:greekLit:tlg5023.tlg011.1st1K-grc1" workUrn="urn:cts:greekLit:tlg5023.tlg011">
+      <ti:label xml:lang="lat">Scholia in Euripidis Rhesum</ti:label>
+      <ti:description xml:lang="mul">Scholia Graeca, Scholia in Euripidis Rhesum, Dindorf, Oxford University Press, 1863</ti:description>
```
Comment (sonofmun) [inline data/tlg5023/tlg011/__cts__.xml]: Change author to "Scholia in Euripidem"

File: data/tlg5023/tlg011/tlg5023.tlg011.1st1K-grc1.xml [hunk 3]
```diff
@@ -0,0 +1,1239 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader xml:lang="eng">
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="lat">Scholia in Euripidis Rhesum</title>
+                <editor>Karl Wilhelm Dindorf</editor>
+                <author xml:lang="lat">Scholia Graeca</author>
```
Comment (sonofmun) [inline data/tlg5023/tlg011/tlg5023.tlg011.1st1K-grc1.xml]: Change author to "Scholia in Euripidem"

File: data/tlg5023/tlg011/tlg5023.tlg011.1st1K-grc1.xml [hunk 4]
```diff
@@ -0,0 +1,1239 @@
...
+                <listBibl>
+                    <biblStruct>
+                        <monogr>
+                            <title xml:lang="lat">Scholia Graeca in Euripidis tragoedias</title>
+                            <editor>
+                                <persName>
+                                    <name xml:lang="lat">Gulielmus Dindorfius</name>
+                                </persName>
+                            </editor>
+                            <author ref="urn:cts:greekLit:tlg5023" xml:lang="lat">Scholia
```
Comment (sonofmun) [inline data/tlg5023/tlg011/tlg5023.tlg011.1st1K-grc1.xml]: Change author to "Scholia in Euripidem"

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |     +102 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +3929 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg5023/__cts__.xml`                          | New      |
| `./data/tlg5023/tlg011/__cts__.xml`                   | New      |
| `./data/tlg5023/tlg011/tlg5023.tlg011.1st1K-grc1.xml` | New      |
| `./data/tlg5023/__cts__.xml`                          | Passing  |
| `./data/tlg5023/tlg011/__cts__.xml`                   | Passing  |
| `./data/tlg5023/tlg011/tlg5023.tlg011.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3428)*

**Pull request merged**